Assets/Atlas games/Scripts/LevelChanges/levelChanges2.cs

```diff
@@ -11,7 +11,14 @@ void Start()
     {
         //GameLevelSetupScr = GameLevelSetupObj.GetComponent<GameLevelSetup>();
 
-        checkWorld();
+        if(GlobalValue.levelPlaying < 150)
+        {
+            wplayMusic();
+        }
+        else if(GlobalValue.levelPlaying > 1000)
+        {
+            endlessPlayMusic();
+        }
```

Comment:
برای ورد های بین 150 تا 1000 چی ؟
```suggestion
        if(GlobalValue.levelPlaying < 1000)
        {
            wplayMusic();
        }
        else // its equivalent to 1000 or greather
        {
            endlessPlayMusic();
        }
```